examples/provider/ThreadedSSLSocketClientServer.java

```diff
@@ -144,7 +144,7 @@ public ThreadedSSLSocketClientServer(String[] args) {
 
         Security.addProvider(new WolfSSLProvider());
 
-        String serverKS = "./examples/provider/server.jks";
+        String serverKS = "./examples/provider/rsa.jks";
```

Comment:
Why do we change the jks here?

Reply:
I had changed this to match the other examples, after seeing a verify error show up with JDK 17.  Have since realized this was due to the example provider keystore/jks files needing to be updated to the current certs.  I switched this back to `server.jks`, and updated the jks files in new commits.